app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedListAdapter.kt

```diff
@@ -60,13 +61,14 @@ class OngoingListAdapter(
       }
       VIEW_TYPE_SECTION_STORY_ITEM -> {
         storyGridPosition = position - titleIndex
-        (holder as OngoingStoryViewHolder).bind(itemList[position] as OngoingStoryViewModel)
+        (holder as PromotedStoryViewHolder)
+          .bind(itemList[position] as PromotedCarouselItemViewModel)
         val marginMin =
           (activity as Context).resources.getDimensionPixelSize(R.dimen.recently_played_margin_min)
         val marginMax =
           (activity as Context).resources.getDimensionPixelSize(R.dimen.recently_played_margin_max)
         val params =
-          holder.binding.ongoingStoryCardView.layoutParams as (ViewGroup.MarginLayoutParams)
+          holder.binding.promotedCarouselItemCardView.layoutParams as (ViewGroup.MarginLayoutParams)
```

Comment:
This ID is incorrect. Try to run the app and then you will see that this is giving error because this id does not exist in xml. Somewhere while renaming this has been renamed incorrectly.